shell/platform/android/io/flutter/embedding/engine/android/AndroidTouchProcessor.java

```diff
@@ -181,7 +183,8 @@ private void addPointerForIndex(
 
     int pointerKind = getPointerDeviceTypeForToolType(event.getToolType(pointerIndex));
 
-    int signalKind = PointerSignalKind.NONE;
+    int signalKind = event.getActionMasked() == MotionEvent.ACTION_SCROLL ?
```

Comment:
Stylistically, can we either keep this on one line, or otherwise break at the "?" and ":"

```java
int signalKind = event.getActionMasked() == MotionEvent.ACTION_SCROLL
  ? PointerSignalKind.SCROLL
  : PointerSignalKind.NONE;
```

Reply:
done